ethereumj-core/src/main/java/org/ethereum/solidity/SolidityType.java

```diff
@@ -329,6 +329,10 @@ public Bytes32Type(String s) {
         public Object decode(byte[] encoded, int offset) {
             return Arrays.copyOfRange(encoded, offset, offset + getFixedSize());
         }
+
+        public static byte[] decodeBytes32(byte[] encoded) {
```

Comment:
Do we get any benefit by adding this method? Maybe just use a simple assignment instead of calling to it?

Reply:
We have similar static methods for other types. So this is just for uniformity

Reply:
Then for uniformity could we add an `offset` param and copy required peace of data, just like it did in [IntType.decodeInt()](https://github.com/ethereum/ethereumj/blob/develop/ethereumj-core/src/main/java/org/ethereum/solidity/SolidityType.java#L402)?